plugin/mlsdbprovider.cpp

```diff
@@ -81,6 +81,8 @@ MlsdbProvider::MlsdbProvider(QObject *parent)
     m_status(StatusUnavailable),
     m_mlsdbOnlineLocator(0),
     m_onlinePositioningEnabled(false),
+    m_signalUpdateCell(false),
```

Comment:
initialisation of these two should occur after initialisation of the m_cellWatcher to avoid a warning